core/services/job/orm.go

```diff
@@ -358,7 +358,7 @@ func (o *orm) FindJobs(offset, limit int) (jobs []Job, count int, err error) {
 			return err
 		}
 
-		sql = `SELECT * FROM jobs ORDER BY id ASC OFFSET $1 LIMIT $2;`
+		sql = `SELECT * FROM jobs ORDER BY created_at, id DESC OFFSET $1 LIMIT $2;`
```

Comment:
Is this something we'd want to document in the changelog to make users aware of it?

Reply:
This is only used by the UI which was doing it's own previously handling the ordering. The problem with this was that it didn't work correctly during pagination.